packages/chrome/src/styled/sheet/StyledSheetTitle.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import {
+  retrieveComponentStyles,
+  getLineHeight,
+  getColor,
+  DEFAULT_THEME
+} from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'chrome.sheet_title';
+
+export const StyledSheetTitle = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<ThemeProps<DefaultTheme>>`
+  margin-bottom: ${props => props.theme.space.base / 4}px;
```

Comment:
This smacks of "pixel nudging" – typically an indication that the design has a bug. Garden never relies on margin nudging like this to match designs. The inherent line height is the only thing that should be managing this space for us.

Reply:
Understood, will update.